### PR TITLE
WL-3466 / WL-3455 : Change wording of Turnitin warning message

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java [hunk 1]
```diff
@@ -8237,7 +8237,9 @@ public void createTIIAssignment(AssignmentContentEdit assign, Time openTime, Tim
             contentReviewService.createAssignment(assign.getContext(), assign.getReference(), opts);
         } catch (Exception e) {
             M_log.error(e);
-            state.setAttribute("alertMessage", rb.getString("content_review.error.createAssignment"));
+            String technicalEmail = ServerConfigurationService.getString("feedback.technicalAddress", null);
```
Comment (buckett) [inline assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java]: Should probably use the "mail.support" configuration variable, as feedback.technicalAddress was introduced by the feedback tool and is currently specific to us.

**Comment by nicholaswilson100** [comment]
ok

**Comment by buckett** [comment]
Doesn't merge cleanly.

Generally it's not ideal to have HTML in i18n strings as it makes it harder for translators and also means they can end up breaking the HTML.

**Comment by nicholaswilson100** [comment]
Rebased back on top.

**Pull request merged**